bpmn-model/src/main/java/io/zeebe/model/bpmn/util/time/TimeDate.java

```diff
@@ -23,9 +27,17 @@ public TimeDate(Interval interval) {
     this.interval = interval;
   }
 
+  public static Timer parse(String timeDate) {
+    final ZonedDateTime dateTime = ZonedDateTime.parse(timeDate);
+    final Interval interval =
+        new Interval(Period.ZERO, Duration.ofMillis(dateTime.toInstant().toEpochMilli()));
+    return new TimeDate(interval);
+  }
+
   @Override
-  public long getDueDate() {
-    return getInterval().toEpochMilli(0);
+  public long getDueDate(long fromEpochMillis) {
+    return getInterval().toEpochMilli(0); // the base milliseconds are ignored since
```

Comment:
I'd add that has a method header, since it's basically describing the whole implementation. But that's just a nitpick, feel free to ignore if you disagree